src/append_only_data.rs

```diff
@@ -206,22 +206,14 @@ impl Permissions for UnpubPermissions {
         self.owner_entry_index
     }
 
-    fn check_permission(&self, requester: Requester, action: Action) -> bool {
-        match requester {
-            Requester::Key(req) => {
-                if self.permissions.contains_key(&req) {
-                    match self.permissions.get(&req) {
-                        Some(perm) => perm.is_allowed(action),
-                        None => false,
-                    }
-                } else {
-                    false
-                }
-            }
-            Requester::Owner(_sig) => {
-                // Temp
-                true
+    fn check_permission(&self, requester: PublicKey, action: Action) -> bool {
+        if self.permissions.contains_key(&requester) {
```

Comment:
Do we need the `if` statement?  Can't we just go straight for `get()`?

Reply:
Good point, thanks :)